### PR TITLE
Add selectors to find in files for selenium tests

### PR DESCRIPTION
**Pull request by melissa-barca**
### Intent

Add selectors to the buttons and table in the `Find in Files` pane so we can access them in unit tests to be added in rstudio/rstudio-ide-automation.

### Approach

Added Class IDs for Fast Select Tables and Left Right Toggle Buttons.

### QA Notes

This shouldn't need any additional QA. We'll know it's working as expected when I submit the unit test.

### CODE REVIEW

File: src/gwt/src/org/rstudio/core/client/widget/FastSelectTable.java [hunk 1]
```diff
@@ -107,6 +110,11 @@ public void onKeyDown(KeyDownEvent event)
          }
       });
    }
+   
+   public void setClassId(String name)
+   {
+      table_.setClassName(ClassIds.FAST_SELECT_TABLE + "_" + name);
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/core/client/widget/FastSelectTable.java:116]: Can this potentially result in class names with spaces in them?

File: src/gwt/src/org/rstudio/core/client/widget/LeftRightToggleButton.java [hunk 2]
```diff
@@ -85,10 +86,21 @@ public LeftRightToggleButton(String leftLabel, String rightLabel,
...
       Roles.getButtonRole().setAriaPressedState(right_, leftIsOn ? PressedValue.FALSE : PressedValue.TRUE);
    }
 
+   public void setClassId()
+   {
+      ClassIds.assignClassId(getElement(),
+         ClassIds.LEFT_RIGHT_TOGGLE_BTN + " _ " +
+            left_.getInnerText() + "_" +
+            right_.getInnerText());
+      ClassIds.assignClassId(left_, ClassIds.LEFT_TOGGLE_BTN + "_"  + left_.getInnerText());
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/core/client/widget/LeftRightToggleButton.java:100]: Same as other question, does this potentially result in class ids with spaces in them?

**Review by gtritchie — CHANGES_REQUESTED** [review]

File: src/gwt/src/org/rstudio/core/client/widget/LeftRightToggleButton.java (hunk 2 above)
Comment (melissa-barca) [reply src/gwt/src/org/rstudio/core/client/widget/LeftRightToggleButton.java]: You're right, thanks! I've fixed the issue.

File: src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputPane.java [hunk 3]
```diff
@@ -134,6 +135,7 @@ public void onKeyUp(KeyUpEvent event)
       FormLabel replaceLabel = new FormLabel("Replace with: ", replaceTextBox_);
       replaceToolbar.addLeftWidget(replaceLabel);
       replaceToolbar.addLeftWidget(replaceTextBox_);
+      Roles.getButtonRole().setAriaLabelProperty(replaceTextBox_.getElement(), "Replace with");
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputPane.java:138]: Why was this added? The replaceTextBox_ already has an accessible label via the above `FormLabel replaceLabel`.

**Review by gtritchie — CHANGES_REQUESTED** [review]

File: src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputPane.java (hunk 3 above)
Comment (melissa-barca) [reply src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputPane.java]: Thanks for pointing this out, I've removed it

**Review by gtritchie — APPROVED** [review]
LGTM!
